pyomo/contrib/alternative_solutions/tests/test_balas.py

```diff
@@ -0,0 +1,145 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright (c) 2008-2024
+#  National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+from collections import Counter
+
+from pyomo.common.dependencies import numpy as numpy, numpy_available
+
+if numpy_available:
+    from numpy.testing import assert_array_almost_equal
+
+import pyomo.environ as pe
+from pyomo.common import unittest
+import pyomo.opt
+
+from pyomo.contrib.alternative_solutions import enumerate_binary_solutions
+import pyomo.contrib.alternative_solutions.tests.test_cases as tc
+
+
+solvers = list(pyomo.opt.check_available_solvers("glpk", "gurobi", "appsi_gurobi"))
+pytestmark = unittest.pytest.mark.parametrize("mip_solver", solvers)
+
+
+@unittest.pytest.mark.default
+class TestBalasUnit:
+
+    def test_ip_feasibility(self, mip_solver):
+        """
+        Enumerate solutions for an ip: triangle_ip.
+
+        Check that there is just one solution when the # of binary variables is 0.
+        """
+        m = tc.get_triangle_ip()
+        results = enumerate_binary_solutions(m, num_solutions=100, solver=mip_solver)
+        assert len(results) == 1
+        assert results[0].objective_value == unittest.pytest.approx(5)
+
+    def Xtest_no_time(self, mip_solver):
```

Comment:
Maybe you could skip the test with a comment that it's fragile? I'm worried with this solution we'll forget about it entirely.

Reply:
Done.